setup.cfg

```diff
@@ -6,7 +6,7 @@ ignore =
     W503
 max-line-length = 99
 statistics = True
-exclude = .venv,venv,build,tutorial,.asv
+exclude = .venv,venv,build,tutorial,.asv,docs/visualization_examples,docs/visualization_matplotlib_examples
```

Comment:
I think it would be great to apply flake8 on visualization example files.

Reply:
In some cases, there are several examples in a single Python file. Multiple import statements appearing in the same file can cause false positives in flake8.

Reply:
Understood. How about splitting example file?